content/docs/rfcs/73/README.md

```diff
@@ -0,0 +1,205 @@
+---
+slug: "73"
+title: 73/WAKU2-INCENTIVIZATION
+name: Incentivization for Waku Light Protocols
+status: raw
+category: Standards Track
+tags:
+  - incentivization
+editor: Sergei Tikhomirov <sergei@status.im>
+contributors:
+---
+# Abstract
+
+This document describes an approach to incentivization of Waku request-response protocols.
+Incentivization is necessary for economically sustainable growth of Waku.
+In an incentivized request-response protocol, only eligible (e.g., paying) clients receive the service.
+Clients include eligibility proofs in their requests.
+
+Eligibility proofs are designed to be used in multiple Waku protocols, such as Store, Lightpush, and Filter.
+Store is planned to become the first Waku protocol to support incentivization.
+We discuss the proof-of-concept implementation of incentivization for Store in a later section.
+
+# Background / Rationale / Motivation
+
+Decentralized protocols require incentivization to be economically sustainable.
+While some aspects of a P2P network can successfully operate in a tit-for-tat model,
+we believe that nodes that run the protocol in good faith need to be tangibly rewarded.
+Motivating servers to expand resources on handling clients' requests allows us to scale the network beyond its initial altruism-based phase.
+
+Incentivization is not necessarily limited to monetary rewards.
+Reputation may also play a role.
+For Waku request-response (i.e., client-server) protocols, we envision a combination of monetary and reputation-based incentivization.
+See a [write-up on incentivization](https://github.com/waku-org/research/blob/1e3ed6a5cc47e6d1e7cb99271ddef9bf38429518/docs/incentivization.md) for our high-level reasoning on the topic.
+
+# Theory / Semantics
```

Comment:
> The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, 
“NOT RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).

@s-tikhomirov also this statement should be placed below "Theory / Semantics"

Reply:
Do you think this paragraph (of a hint to include this paragraph) should also be added to the [RFC template](https://github.com/vacp2p/rfc/blob/master/content/docs/rfcs/template/README.md)?

Reply:
@s-tikhomirov Yes, we are currently working on updating [1/COSS](https://github.com/vacp2p/rfc/tree/master/content/docs/rfcs/1) and the [RFC template](https://github.com/vacp2p/rfc/blob/master/content/docs/rfcs/template/README.md)